### ISSUE TITLE
use this

### ISSUE THREAD

**Issue by joelmccracken** [joelmccracken/cr_buddy#1]
https://github.com/octokit/octokit.rb

**Comment by machuga** [comment]
^^ I use that.  Highly recommend.  Hypermedia! \o/

**Comment by joelmccracken** [comment]
Yeah. I searched Ma Alphabet for github API gem, but the results weren't good. Best I found is what is used in the gem file.
